main.py

```diff
@@ -75,12 +52,9 @@ def shutdown(self):
 
         If you have some cleanup procedure, insert it here.
         """
-        event = KytosEvent('amlight/scheduler.remove_job')
-        event.content['id'] = 'automatic_traces'
-        self.controller.buffers.app.put(event)
-        event = KytosEvent('amlight/scheduler.remove_job')
-        event.content['id'] = 'automatic_important_traces'
-        self.controller.buffers.app.put(event)
+        id_set = {'automatic_traces', 'automatic_important_traces'}
```

Comment:
This should work but to make it less susceptible for developers to keep updating this set here if new jobs are scheduled in the future, you can iterate on `self.automate.ids` that you've already storing the scheduled values, could you update this?

Reply:
Got it, added a default value to `automate.unschedule_ids()` so it can iterate `self.automate.ids` or a passing argument.